ietf/doc/views_doc.py

```diff
@@ -494,7 +594,7 @@ def document_main(request, name, rev=None, document_html=False):
 
         augment_docs_and_user_with_user_info([doc], request.user)
 
-        published = doc.latest_event(type="published_rfc")
+        published = doc.latest_event(type="published_rfc")  # todo rethink this now that published_rfc is on rfc
```

Comment:
Correct - this should always be None for a document of type draft. If we want the draft view to say anything about when the rfc it became was published we should be looking through the relationship.